x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -76,6 +77,13 @@ const getCommentContent = (comment: CommentResponse): string => {
   } else if (comment.type === CommentType.alert || comment.type === CommentType.generatedAlert) {
     const ids = getAlertIds(comment);
     return `Alert with ids ${ids.join(', ')} added to case`;
+  } else if (comment.type === CommentType.actions) {
+    const firstHostname = comment.actions.targets[0]?.hostname;
+    const totalHosts = comment.actions.targets.length;
+    const actionText = comment.actions.type === 'isolate' ? 'Isolated' : 'Released';
+    const additionalHostsText = totalHosts - 1 > 0 ? `and ${totalHosts - 1} more ` : ``;
+
+    return `${actionText} host ${firstHostname} ${additionalHostsText}with comment: ${comment.comment}`;
```

Comment:
Could you add a test or two in https://github.com/elastic/kibana/blob/master/x-pack/plugins/cases/server/client/cases/utils.test.ts#L203 just to ensure the string is getting created like we expect.